### PR TITLE
[FLINK-11304][docs][table] Fix typo in time attributes doc

### PR DESCRIPTION
**Pull request by JaryZhen**
## What is the purpose of the change

This pull request fix typo in  time attributes docs  **DefinedRowtimeAttribute** should be **DefinedRowtimeAttributes**

## Brief change log

fix typo in  time attributes doc

## Verifying this change
This change is a trivial rework / code cleanup without any test coverage

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no

### CODE REVIEW

**Comment by JaryZhen** [comment]
Here I talk about is **DefinedRowtimeAttribute** should be **DefinedRowtimeAttributes**

**Comment by KarmaGYZ** [comment]
Sorry for the misunderstanding @JaryZhen .
I think we should also change the `getRowtimeAttribute` to `getRowtimeAttributeDescriptors` as well and reimplement the method.
By the way, since you create a new ticket in this issue, we can change to PR title, start with [FLINK-11304].

**Comment by JaryZhen** [comment]
OK,  I do it right now.

File: docs/dev/table/streaming/time_attributes.md [hunk 1]
```diff
@@ -284,9 +284,9 @@ public class UserActionSource implements StreamTableSource<Row>, DefinedRowtimeA
 	}
 
 	@Override
-	public String getRowtimeAttribute() {
-		// Mark the "UserActionTime" attribute as event-time attribute.
-		return "UserActionTime";
+	public List[RowtimeAttributeDescriptor] getRowtimeAttributeDescriptors() {
```
Comment (KarmaGYZ) [inline docs/dev/table/streaming/time_attributes.md]: Should be `List<RowtimeAttributeDescriptor>`

File: docs/dev/table/streaming/time_attributes.md [hunk 2]
```diff
@@ -284,9 +284,9 @@ public class UserActionSource implements StreamTableSource<Row>, DefinedRowtimeA
...
+		// Mark the "UserActionTime" attribute as event-time attribute.
```
Comment (KarmaGYZ) [inline docs/dev/table/streaming/time_attributes.md]: Remove the blank in the end.

File: docs/dev/table/streaming/time_attributes.md [hunk 3]
```diff
@@ -284,9 +284,9 @@ public class UserActionSource implements StreamTableSource<Row>, DefinedRowtimeA
 	}
 
 	@Override
-	public String getRowtimeAttribute() {
-		// Mark the "UserActionTime" attribute as event-time attribute.
-		return "UserActionTime";
+	public List[RowtimeAttributeDescriptor] getRowtimeAttributeDescriptors() {
+		// Mark the "UserActionTime" attribute as event-time attribute. 
+		// return List<RowtimeAttributeDescriptor>;
```
Comment (KarmaGYZ) [inline docs/dev/table/streaming/time_attributes.md]: I think we should implement it in doc.

File: docs/dev/table/streaming/time_attributes.md [hunk 4]
```diff
@@ -317,9 +317,9 @@ class UserActionSource extends StreamTableSource[Row] with DefinedRowtimeAttribu
 		stream
 	}
 
-	override def getRowtimeAttribute = {
+	override def getRowtimeAttributeDescriptors: util.List[RowtimeAttributeDescriptor] = {
 		// Mark the "UserActionTime" attribute as event-time attribute.
-		"UserActionTime"
+		// List<RowtimeAttributeDescriptor>
```
Comment (KarmaGYZ) [inline docs/dev/table/streaming/time_attributes.md]: Same as above.

**Comment by JaryZhen** [comment]
I don't know why this happened while in doc is right but in **md** not
image

**Comment by KarmaGYZ** [comment]
@JaryZhen You should use tab, not blank. :)

**Comment by JaryZhen** [comment]
@KarmaGYZ Thanks, I fixed that :)

File: docs/dev/table/streaming/time_attributes.md [hunk 5]
```diff
@@ -284,13 +284,19 @@ public class UserActionSource implements StreamTableSource<Row>, DefinedRowtimeA
 	}
 
 	@Override
-	public String getRowtimeAttribute() {
+	public List<RowtimeAttributeDescriptor> getRowtimeAttributeDescriptors() {
 		// Mark the "UserActionTime" attribute as event-time attribute.
-		return "UserActionTime";
+		// here we create one attribute descriptor of "UserActionTime"
+		RowtimeAttributeDescriptor rowtimeAttrDescr = new RowtimeAttributeDescriptor(
+		    "UserActionTime",
```
Comment (KarmaGYZ) [inline docs/dev/table/streaming/time_attributes.md]: Still using blank.

File: docs/dev/table/streaming/time_attributes.md [hunk 6]
```diff
@@ -317,9 +323,15 @@ class UserActionSource extends StreamTableSource[Row] with DefinedRowtimeAttribu
 		stream
 	}
 
-	override def getRowtimeAttribute = {
+	override def getRowtimeAttributeDescriptors: util.List[RowtimeAttributeDescriptor] = {
 		// Mark the "UserActionTime" attribute as event-time attribute.
-		"UserActionTime"
+		// here we create one attribute descriptor of "UserActionTime"
+		val rowtimeAttrDescr = new RowtimeAttributeDescriptor(
+		    "UserActionTime",
```
Comment (KarmaGYZ) [inline docs/dev/table/streaming/time_attributes.md]: Same as above.

File: docs/dev/table/streaming/time_attributes.md (hunk 5 above)
Comment (JaryZhen) [reply docs/dev/table/streaming/time_attributes.md]: Hi, I think it's correct now, here is the code view form my patch.
image

File: docs/dev/table/streaming/time_attributes.md (hunk 6 above)
Comment (JaryZhen) [reply docs/dev/table/streaming/time_attributes.md]: this too
image

Comment (JaryZhen) [reply docs/dev/table/streaming/time_attributes.md]: @KarmaGYZ pls tell me if there is anything need be improve, Thanks.

Comment (KarmaGYZ) [reply docs/dev/table/streaming/time_attributes.md]: @JaryZhen I still prefer to use tab. Besides, I think you should push this commit the master branch.

**Pull request closed without merging**